server/plugins/LFUGens.cpp

```diff
@@ -897,7 +897,9 @@ void LFGauss_Ctor(LFGauss* unit)
 	}
 	unit->mPhase = -1.0;
 
-	//LFGauss_next_k(unit, 1);
```

Comment:
Only @telephon would know why, if he remembers. It was introduced in the initial commit for LFGauss in 9bfb7474.

I could have simply uncommented it, but that would be a behavior change.